pyproject.toml

```diff
@@ -39,7 +39,9 @@ dependencies = [
   "numpy",
   "fsspec",
   "packaging",
-  "typing_extensions>=4.1.0; python_version < \"3.11\""
+  "typing_extensions>=4.1.0; python_version < \"3.11\"",
+  "dask >=2023.04.0",
+  "dask-awkward>=2023.12.1"
```

Comment:
dask shouldn't be a top level dependency of the uproot package? @jpivarski 

I suppose below we should import dask within `dask_write` such that it complains when dask is not available, but doesn't impede more standard functioning of uproot.

Reply:
Correct: `dask` and `dask-awkward` should not be strict dependencies for Uproot. These two lines should be removed.